utils/clusterTester/columnstoreClusterTester.sh

```diff
@@ -667,12 +669,12 @@ checkFirewalls()
       # 'sysconfig not on remote node
       for firewall in "${FIREWALL_LIST[@]}"; do
 	pass=true
-        `$COLUMNSTORE_INSTALL_DIR/bin/remote_command.sh $ipadd $PASSWORD "service '$firewall' status > /tmp/firewall_check 2>&1" 1 > /tmp/remote_command_check`
+        `$COLUMNSTORE_INSTALL_DIR/bin/remote_command.sh $ipadd $PASSWORD "service '$firewall' status > ${tmpDir}/firewall_check 2>&1" 1 > ${tmpDir}/remote_command_check`
         if [ "$?" -eq 0 ]; then
               echo "${bold}Warning${normal}, $ipadd Node $firewall service is Active, check port test results"
               pass=false
         else
-	  `$COLUMNSTORE_INSTALL_DIR/bin/remote_command.sh $ipadd $PASSWORD "systemctl status '$firewall' > /tmp/firewall_check 2>&1" 1 > /tmp/remote_command_check`
+	  `$COLUMNSTORE_INSTALL_DIR/bin/remote_command.sh $ipadd $PASSWORD "systemctl status '$firewall' > ${tmpDir}/firewall_check 2>&1" 1 > ${tmpDir}/remote_command_check`
 	  if [ "$?" -eq 0 ]; then
 	      echo "${bold}Warning${normal}, $ipadd Node $firewall service is Active, check port test results"
 	      pass=false
```

Comment:
are we failing a config check b/c a firewall exists?  Can we replace these checks with attempts to connect on CS ports instead?